tools/aad.sh

```diff
@@ -35,6 +36,27 @@ app-create)
         ]
     }
 ]
+EOF
+)
+
+    az ad sp create --id $AZURE_CLIENT_ID >/dev/null
+
+    success=
+    for ((i=0; i<12; i++)); do
```

Comment:
racing something here?

Reply:
yes :(

`Principal 38e037faaa2c4aabbabc1d8f3532a618 does not exist in the directory 7a1815fb-63ba-4af7-8f33-e7333e421980.`

See https://github.com/Azure/azure-powershell/issues/2286 for example.